tools/windows/DatadogAgentInstaller/CustomActions/InstallOciPackages.cs

```diff
@@ -30,44 +29,59 @@ public InstallOciPackages(ISession session)
             _rollbackDataStore = new RollbackDataStore(session, "InstallOciPackages", new FileSystemServices(), new ServiceController());
         }
 
-
-        private string PackageName(string language)
+        private bool ShouldPurge()
         {
-            return $"datadog-apm-library-{language}";
+            var purge = _session.Property("PURGE");
```

Comment:
If we don't have a reason not to since our last discussion, I'm in favor of making purge the default and skipping it only when a flag is passed.